tests/CMakeLists.txt

```diff
@@ -16,9 +16,6 @@ if(QUDA_QDPJIT)
 endif()
 
 if(QUDA_QIO AND QUDA_DOWNLOAD_USQCD)
```

Comment:
Similar comment as above, not sure what this has to do with this PR, though I can see that it's important for functional correctness. @maddyscientist what do you think?

Reply:
If this is going wrong we should fix it. Not quite sure removing the `if(QUDA_QMP)` is the way to fix it as we can have only QMP enabled.

Reply:
@mathiaswagner for sure, agreed. Ultimately my comment is a splitting-hairs comment of if we want to batch it in with this PR or not. I guess it doesn't make a difference.

Reply:
I know and I technically agree that it might be cleaner to have a separate PR. Ultimately we just need to make sure that the fix is actually the proper fix.